src/style/theme.scss

```diff
@@ -15,6 +15,7 @@ $iui-color-background-disabled--light: $iui-color-background-3--light;
 
 // Light theme > Foreground
 $iui-color-foreground-body--light: #000;
+$iui-color-foreground-body-invert--light: #FFF;
```

Comment:
Where is all this supposed to be used?

Reply:
I do not have a need for these variables at this moment.  If we were to have the tooltips switch with the theme, these would be useful in that scenario.  @DanEastBentley would like to use these variables for the text overlaying an icon (screenshots attached).
<img width="293" alt="Screen Shot 2021-04-22 at 4 29 41 PM" src="https://user-images.githubusercontent.com/849817/115781267-ffe72300-a36e-11eb-8d55-de4efd765c47.png">